src/handlers/decision.rs

```diff
@@ -152,6 +162,119 @@ pub(super) async fn handle_command(
     }
 }
 
+fn build_status_comment(
+    history: &BTreeMap<String, Vec<UserStatus>>,
+    current: &BTreeMap<String, Option<UserStatus>>,
+) -> anyhow::Result<String> {
+    let mut comment = "| Team member | State |\n|-------------|-------|".to_owned();
+    for (user, statuses) in history {
+        let mut user_statuses = format!("\n| {} |", user);
+
+        // previous stasuses
+        for status in statuses {
+            let status_item = format!(" ~~{}~~ ", resolution_to_str(&status.resolution));
+            user_statuses.push_str(&status_item);
+        }
+
+        // current status
+        let current_status = current.get(user).unwrap();
+        let user_resolution;
+        match current_status {
+            Some(status) => user_resolution = resolution_to_str(&status.resolution),
+            _ => user_resolution = "".to_string(),
+        }
+        let status_item = format!(" **{}** |", user_resolution);
+        user_statuses.push_str(&status_item);
+
+        comment.push_str(&user_statuses);
+    }
+
+    println!("{}", comment);
+    Ok(comment)
+}
+
+fn resolution_to_str(resolution: &Resolution) -> String {
```

Comment:
What do you think about implementing [Display](https://doc.rust-lang.org/std/fmt/trait.Display.html) for `Resolution` instead of doing this way?

Reply:
Like the idea!
I also though of doing something like:

```rs
impl Resolution {
    pub fn to_str(&self) -> String {
        match self {
            Resolution::Merge => "merge".to_owned(),
            Resolution::Hold => "hold".to_owned(),
        }
    }
}
```

But it would have to return a `Result` since it can fail (and right now I'm just returning an empty string which is bad)
So I'll do `fmt::Display`!